File: July 2024/July 10th/beautifulDay.js
function beautifulDays(firstDay, lastDay, k) {
    let beauDays = 0
    
    for(let i = firstDay; i<=lastDay; i++){
        let numStr = String(i)
        let dayArr = numStr.split("")
        let arrSwap = dayArr.reverse()
        let daySwap = arrSwap.join("")
        
        if((i-daySwap)%k===0){
            beauDays++
        }
    }

    return beauDays
}

File: July 2024/July 10th/beautifulDay.py
def beautifulDays(firstDay, lastDay, k):
    beauDays = 0
    i = firstDay
    while i <= lastDay:
        numStr = str(i)
        dayList = list(numStr)
        dayList.reverse()
        convertedList = map(str, dayList)
        daySwap = ''.join(convertedList)

        if (i - int(daySwap))%k == 0:
            beauDays += 1

        i += 1

    return beauDays

print(beautifulDays(20,23,6))